starlette/testclient.py

```diff
@@ -381,13 +381,11 @@ def receive_json(self, mode: str = "text") -> typing.Any:
 
 class TestClient(requests.Session):
     __test__ = False  # For pytest to not discover this up.
-
-    #: These options are passed to `anyio.start_blocking_portal()`
+    #: These are the default options for the constructor arguments
```

Comment:
You should be able to move the type annotation onto the assignment in `def __init__`

Reply:
Oh I see this is a ClassVar